routes/sites.js

```diff
@@ -118,7 +118,52 @@ async function checkHasAccess (req, res, next) {
   }
 }
 
+/* Gets the last updated time of the repo. */
+async function getLastUpdated (req, res, next) {
+  const { accessToken } = req
+  const { siteName } = req.params
+
+  const endpoint = `https://api.github.com/repos/${ISOMER_GITHUB_ORG_NAME}/${siteName}`
+  const resp = await axios.get(endpoint, {
+    headers: {
+      Authorization: `token ${accessToken}`,
+      "Content-Type": "application/json",
+    }
+  })
+  const { updated_at } = resp.data
+  res.status(200).json({ lastUpdated: timeDiff(updated_at)})
+}
+
+/* Gets the link to the staging site for a repo. */
+async function getStagingUrl (req, res, next) {
+  // TODO: reconsider how we can retrieve url - we can store this in _config.yml or a dynamodb
+  const { accessToken } = req
+  const { siteName } = req.params
+  
+  const endpoint = `https://api.github.com/repos/${ISOMER_GITHUB_ORG_NAME}/${siteName}`
+  const resp = await axios.get(endpoint, {
+    headers: {
+      Authorization: `token ${accessToken}`,
+      "Content-Type": "application/json",
+    }
+  })
+
+  const { description } = resp.data
+
+  let stagingUrl
+
+  if (description) {
+    // Retrieve the url from the description - repo descriptions have varying formats, so we look for the first link
+    const descTokens = description.replace('/;/g', ' ').split(' ')
+    stagingUrl = descTokens.find(token => token.includes('http'))
```

Comment:
would it be good to also check for `token.includes("staging")`?

Reply:
Great point, resolved in https://github.com/isomerpages/isomercms-backend/pull/142/commits/4e208f22549187f36c40f878a1858e30667eafb9